api/src/basic.rs

```diff
@@ -60,6 +65,56 @@ impl BasicApi {
         Html(OPEN_API_HTML.to_string())
     }
 
+    /// Show some basic info of the node.
+    #[oai(
+        path = "/info",
+        method = "get",
+        operation_id = "info",
+        tag = "ApiTags::General"
+    )]
+    async fn info(&self) -> Json<HashMap<String, String>> {
```

Comment:
This output will be more standard and parseable if you just use JSON values. How about you instead use:
```
Json<HashMap<String, serde_json::Value>>
```
as the output?

As it is now you're using the debug representation inside a string, so that will be difficult to parse. Like it'll look sort of like JSON but not quite.

So instead of:
```
            format!(
                "{:?}",
                self.context
                    .node_config
                    .state_sync
                    .state_sync_driver
                    .bootstrapping_mode
            ),
```

Do this:
```

                serde_json::Value::from(&self.context
                    .node_config
                    .state_sync
                    .state_sync_driver
                    .bootstrapping_mode)
```

Reply:
Even better, you could also just return `Json<HashMap<String, T>> where `T: Serialize` and then poem will do the serialization for you.

Reply:
Done